usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -99,7 +99,7 @@ def handle(self, *args, **options):
         loader.load_from_file(options['file'][0])
 
     def get_agency(self, agency_string):
-        agency = Agency.objects.filter(subtier_code=self.get_agency_code(agency_string)).first()
+        agency = Agency.objects.filter(subtier_agency__subtier_code=self.get_agency_code(agency_string)).first()
```

Comment:
I do believe they are, they are based off of cgac_code if I am not mistaken, so a subtier inside of something with a cgac_code of 13 will always start with 13. i.e. 1341 is NIST and 1300 is DOC